File: 01_Basico/01 - Conceitos_Basicos/Respostas_em_Python/questao07.py
preco_original = float(input("Digite o valor do preço do produto"))
porcentagem_desconto = 20
desconto = (preco_original * porcentagem_desconto) / 100
print(f"Valor original:             R${preco_original}")
print(f"Desconto ganho:             R${desconto}")
print(f"O Valor comm desconto é:    R$ {preco_original - desconto}")

File: 01_Basico/01 - Conceitos_Basicos/Respostas_em_Kotlin/questao07.kt
fun main() {
    val produto = 100f
    val porcentagem_desconto = 20f
    val desconto = (produto * porcentagem_desconto) / 100
    println("O valor original:		R$ $produto")
  	println("Desconto ganho:			R$  $desconto")
    println("O valor com desconto é:		R$  ${produto - desconto}")
}